.github/workflows/icon-review.yml

```diff
@@ -1,23 +1,23 @@
 name: 🏞️ Icon Review
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
Why did you change this?

Reply:
zizmor - https://github.com/material-extensions/vscode-material-icon-theme/issues/2796